Gemini/README.md

```diff
@@ -0,0 +1,95 @@
+# Gemini 
```

Comment:
@manez better be prepared.  Each one of these needs a doe-eyed avatar.  Are we prepared to maintain all these mascots?